core/src/test/java/org/apache/iceberg/TestMetadataUpdateParser.java

```diff
@@ -21,11 +21,33 @@
 
 import java.util.List;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
+import org.apache.iceberg.types.Types;
+import org.assertj.core.api.Assertions;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class TestMetadataUpdateParser {
 
+  // MetadataUpdate actions
```

Comment:
Can you make the other references package-private so you don't have to copy them? Or is this for testing purposes so that you ensure you're not altering tests by altering the constants? (I probably wouldn't worry about that)

Reply:
package-private would be fine. The consideration around not altering constants was not something I'd thought of.